crypto/ube/ube_test.cc

```diff
@@ -62,8 +101,34 @@ TEST(Ube, MockedMethodTests) {
   ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
   ASSERT_EQ(generation_number, cached_generation_number);
 
-  // The snapsafe generation number is initially 1. Again use 10.
-  set_snapsafe_generation_number_FOR_TESTING(10);
+  // Mock another fork event but with a strictly smaller value.
+  set_snapsafe_generation_number_FOR_TESTING(5);
+
+  // Generation number should have incremented once.
+  cached_generation_number = generation_number;
+  generation_number = 0;
+  ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
+  ASSERT_EQ(generation_number, cached_generation_number + 1);
+
+  // Should be stable again.
+  cached_generation_number = generation_number;
+  generation_number = 0;
+  ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
+  ASSERT_EQ(generation_number, cached_generation_number);
+
+  // Now test incrementing the snapsafe generation number. Before we used 10 as
+  // the default value, use a smaller number now, say 5.
+  set_snapsafe_generation_number_FOR_TESTING(5);
+  ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
```

Comment:
I'm a little confused, isn't the generation number already 5 from line 105?

Reply:
I accidentally added this twice.

Reply:
deleted this and the following stanza